### ISSUE TITLE
Should I update my .m3u each time there is an update from your side?

### ISSUE THREAD

**Issue by filoppan** [free-greek-iptv/greek-iptv#36]
Many thanks!
Panos

**Comment by stav8124** [comment]
> I use it on my LG smart TV, via the app called HomeIPTV
> Many thanks!
> Panos

I use SmartClub.tv on LG Smart Tv but having issues with EPG working.  Does EPG work on yours on HomeIPTV?

**Comment by filoppan** [comment]
> > I use it on my LG smart TV, via the app called HomeIPTV
> > Many thanks!
> > Panos
> 
> I use SmartClub.tv on LG Smart Tv but having issues with EPG working. Does EPG work on yours on HomeIPTV?

Not really. I get the EPG as a setting, but when I click it, nothing happens unfortunately
